Modules/Loadable/Segmentations/EditorEffects/Python/SegmentEditorEffects/SegmentEditorMaskVolumeEffect.py

```diff
@@ -31,10 +34,10 @@ def icon(self):
         return qt.QIcon()
 
     def helpText(self):
-        return """<html>Use the currently selected segment as a mask to blank out regions in a volume.
+        return "<html>" + _("""Use the currently selected segment as a mask to blank out regions in a volume.
 <br> The mask is applied to the source volume by default.<p>
 Fill inside and outside operation creates a binary labelmap volume as output, with the inside and outside fill values modifiable.
```

Comment:
```suggestion
Fill inside and outside operation creates a binary labelmap volume as output, with the inside and outside fill values modifiable.</p>
```

Reply:
I would simply add `<p>` here for keeping the text simpler.